backend/order/migrations/0001_initial.py

```diff
@@ -34,4 +35,15 @@ class Migration(migrations.Migration):
                 ('order', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='order.order')),
             ],
         ),
+        migrations.CreateModel(
+            name='FeedbackAndRating',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('rating', models.PositiveIntegerField(blank=True, choices=[(1, 1), (2, 2), (3, 3), (4, 4), (5, 5)], default=1, null=True, validators=[django.core.validators.MaxValueValidator(5)])),
```

Comment:
@julkascript here the choices are transformed to array of tuples with just number.